### DISCUSSION TITLE
Maximum execution time of 120 seconds exceeded

### DISCUSSION THREAD

**Discussion by aityomin** [totumonline/totum-ru-issues-and-discussions#436] [Неопределенное]
День добрый! Чем это может быть вызвано?
<b>Fatal error</b>:  Maximum execution time of 120 seconds exceeded in <b>/home/totum/totum-mit/totum/tableTypes/aTable.php</b> on line <b>1284</b><br />
Это происходит в таблице циклов.

**Comment by ttm-alexeypolunin** [comment]
Это не ошибка платформы — просто вы написали цепочку кодов и действий, которая не успевает выполнится за лимит 120 секунд.
